car-park.swagger.yaml

```diff
@@ -90,31 +67,62 @@ definitions:
         $ref: '#/definitions/RealtimeStatus'
     required:
       - identifier
-      - lat
-      - lng
+      - coordinates
       - name
       - address
 
-  TimePeriod:
-    description: Information about time periods
+  OpeningHours:
     type: string
+    description: Opening times
+    externalDocs:
+     url: "http://schema.org/openingHours"
     example:
-      - "24/7" # all week
+      - "Mo-Su" # all week
       - "Mo-Fr" # all day from Mondays to Fridays
-      - "08:00-21:00" # all week from 8:00 to 21:00
-      - "Mo-Fr 06:30-21:00; Sa 08:00-14:00"  # a very descriptive example
+      - "Mo-Su 08:00-21:00" # all week from 8:00 to 21:00
+      - "Tu,Th 08:00-12:00" # Tuesdays and Thursdays from 8:00 to 12:00
+      - ["Mo-Fr 06:30-21:00", "Sa 08:00-14:00"]  # using an array for a complex one
```

Comment:
The last example doesn't match the schema. Can you update the schema accordingly? What we'd want is that `OpeningHours` uses `oneOf`, which is then either a string or an array of strings. Need to switch to Open API 3.0 for that though. See https://openapi-converter.herokuapp.com

Reply:
Though that's poor API design. How about we just make this an array of `OpeningHours` strings instead wherever it's being used? And usually it's just a single string in the array? Makes the API simpler to use and parse.

Reply:
> How about we just make this an array of `OpeningHours` strings instead wherever it's being used? 
Yes. I would prefer this. Much easier to use and parse.